tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -39,6 +39,13 @@ internal static class Debugger
             /// <seealso cref="DebuggerSettings.UploadBatchSize"/>
             public const string UploadBatchSize = "DD_DEBUGGER_UPLOAD_BATCH_SIZE";
 
+            /// <summary>
+            /// Configuration key for the maximum symbol size to upload (in mb).
+            /// Default value is 1 mb.
+            /// </summary>
```

Comment:
The comment seem to be incorrect. From the code it's apparent that you're using byte as the unit.